aten/src/ATen/native/native_functions.yaml

```diff
@@ -3158,6 +3158,19 @@
   dispatch:
     CPU, CUDA: trunc_out
 
+# Alias of trunc
+- func: fix(Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  
+- func: fix_(Tensor(a!) self) -> Tensor(a!)
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: fix.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
```

Comment:
Is the explicit dispatch here (and for `trunc.out`) necessary?

Reply:
I did some short tests and seems it isn't, hopefully CI passes.